src/server.rs

```diff
@@ -600,6 +600,26 @@ impl<C: CommandCreatorSync> SccacheServer<C> {
     }
 }
 
+/// entry of the compiler cache
+struct CompilerCacheEntry<C : CommandCreatorSync> {
+    /// compiler argument trait obj
+    pub compiler : Box<dyn Compiler<C>>,
+    /// when did this entry get inserted into the cache
+    pub mtime : FileTime,
```

Comment:
Is this comment correct?  I would expect this field to be related to the modification time of `compiler` in some fashion.

Reply:
You are correct, the comment is wrong.